Add documentation for Manage Core Configuration (#1156)
auth0/docs-v2: main/docs/customize/login-pages/manage-core-configuration.mdx

---
title: "Manage Core Configuration"

> description: "Manage your login engine, WCAG compliance, and virtual environment support."

Reviewer: ```suggestion
description: "Manage your login engine, WCAG compliance, and iframe embedding settings."
```

---

> Configure your Universal or Classic Login's core configuration, which includes:

Reviewer: ```suggestion
The Core configuration page lets you manage the following settings for Universal Login and Classic Login:
```

- **Login engine**: Switch between Universal and Classic Login.

> - **Web Content Accessibility Guidelines (WCAG) compliance**: Create a beautiful, hosted login page that meets WCAG requirements.

Reviewer: no guidance here on this?

Author: i included it because it's in the Dashboard, but let me adjust to make it clearer that with Universal Login, you don't have to do anything additional for it to meet WCAG compliance

> - **Web Content Accessibility Guidelines (WCAG) compliance**: Create a beautiful, hosted login page that meets WCAG requirements.

Reviewer: ```suggestion
- **Web Content Accessibility Guidelines (WCAG) compliance**: Configure your hosted login page to meet WCAG requirements.
```
I'm sure it _will_ be beautiful, but idk about that verbiage here.

Author: you're so right i just used what was in the Dashboard lmao

> - **Virtual environment support**: Enable iframe embedding for development environments.

Reviewer: ```suggestion
- **Iframe embedding**: Allow login flows to load inside iframes for development environments."
```
keeping terminology consistent with section headings

Author: it says virtual environment support in the UI, so i'm going to leave the colon as is

## Settings

### Universal or Classic Login

> Switch between Universal and Classic Login. To learn more, read [Universal Login vs. Classic Login](/docs/customize/login-pages).

Reviewer: ```suggestion
Switch between Universal and Classic Login. To learn more, read [Universal Login vs. Classic Login](/docs/authenticate/login/auth0-universal-login/universal-login-vs-classic-login).
```

Reviewer: i think this is a better destination?

> Switch between Universal and Classic Login. To learn more, read [Universal Login vs. Classic Login](/docs/customize/login-pages).

Reviewer: ```suggestion
Choose which login experience your tenant uses. Changing this setting affects all applications that redirect to your Auth0 login page. To learn more, read [Universal Login vs. Classic Login](/docs/customize/login-pages).
```
beefing this up a bit to not just restate the heading

<Steps>
<Step title="Navigate to Core configuration">
Navigate to **Branding > Universal Login > Pro-code customizations** and select **Core configuration**.
</Step>

<Step title="Select login type">
Under **Settings**, select **Universal Login** or **Classic Login**.
</Step>
</Steps>

### Iframe embedding

> Enable Universal or Classic Login to load when the application and login flow are embedded in an iframe, as with generative UI platforms like Vercel.

Reviewer: Vercel isn't really a generative UI platform, unless you're talking specifically about v0?

Author: the toggle itself is "Enable support for generative UI platforms," so i'm going to leave as is

<Warning>
When iframe embedding is enabled, security headers that protect against clickjacking are disabled.
</Warning>

> <Warning>

Reviewer: Back to back warnings here doesn't look great. can you combine them?

When enabled, iframe embedding is only applied in development environments.
</Warning>

<Steps>
<Step title="Navigate to Core configuration">
Navigate to **Branding > Universal Login > Pro-code customizations** and select **Core configuration**.
</Step>

<Step title="Enable iframe support">
Under **Iframe settings**, toggle on **Enable support for generative UI platforms**.
</Step>

<Step title="Configure allowed origins">

> Under **Allowed iframe URLs for Login**, enter the allowed origins (e.g., `https://v0.app` or `https://*.vusercontent.net`).

Reviewer: ```suggestion
Under **Allowed iframe URLs for Login**, enter the allowed origins (for example, `https://v0.app` or `https://*.vusercontent.net`).
```
no latin abbreviations

> When an application is created from a generative UI platform such as Vercel using an available Auth0 integration, Auth0 pre-populates the allowed origins from the platform. To learn more, read [Integrate with Vercel](/docs/customize/integrations/integrate-with-vercel).

Reviewer: Bringing up the generative UI comment again here

Author: see above
